plugins/GSdx/GS.cpp

```diff
@@ -515,7 +510,7 @@ EXPORT_C_(int) GSopen(void** dsp, char* title, int mt)
 	XCloseDisplay(display);
 	*/
 
-	int renderer = 0;
+	GS_RENDERER renderer = GS_RENDERER::DX9_HW;
```

Comment:
Possibly this should be GS_DEFAULT_RENDERER instead. I was just translating here and didn't want to change actual code. But this line shouldn't make sense for linux.